main.go

```diff
@@ -235,6 +236,26 @@ func init() {
 		}
 	}
 
+	if config.Quickwit.HostPort != "" {
+		var err error
+
+		if config.Quickwit.ApiEndpoint == "" {
+			config.Quickwit.ApiEndpoint = "/api/v1"
+		}
+
+		if config.Quickwit.Index == "" {
+			config.Quickwit.Index = "falco"
```

Comment:
@fmassot which default name of index would you pick? I thought you'd like a default index the same way for the otel logs and traces.

Reply:
do we have an idea of the format of a document generated by falco?

Reply:
Yes, here's the go representation of the payload:

```go
type FalcoPayload struct {
	UUID         string                 `json:"uuid,omitempty"`
	Output       string                 `json:"output"`
	Priority     PriorityType           `json:"priority"`
	Rule         string                 `json:"rule"`
	Time         time.Time              `json:"time"`
	OutputFields map[string]interface{} `json:"output_fields"`
	Source       string                 `json:"source"`
	Tags         []string               `json:"tags,omitempty"`
	Hostname     string                 `json:"hostname,omitempty"`
}

type PriorityType int
```

Reply:
`falco` as default name is ok.

For the event format, you can use my script to generate fake events, I created it for my tests with falcosidekick/ui 

https://gist.github.com/Issif/8236641b072d4ef409a3ceac899fb02c

Reply:
I also think about make a POST request to create the mapping at the init if the index doesn't exists.

Reply:
Creating a mapping could be nice indeed to quick start, maybe could be worth to have a parameter like `auto-create-index` to enable/disable this?
In the mapping, you could only declare the time field and the field on which we can do full text search.